### PR TITLE
KristenMcWilliam.Nyrna version 2.1.0

### PR DESCRIPTION
**Pull request by Merrit**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25052)

### CODE REVIEW

**Review by ImJoakim — CHANGES_REQUESTED** [review]
The DisplayVersion is `2.0.0-beta.2`. If you want to upload this specific package, you would need to change the version to that. However, I see that you're the person who released it, so maybe you are able to update the DisplayVersion to `2.1.0` (which I'm guessing it's supposed to be) and you won't have to call it `2.0.0-beta.2`.

image

**Comment by ImJoakim** [comment]
And the DisplayName:

image

**Comment by Merrit** [comment]
@ImJoakim Nice catch, thank you!

I've updated my CI pipeline to properly forward the current version info for both variables to inno setup, and updated the binary at the download url.

File: manifests/k/KristenMcWilliam/Nyrna/2.1.0/KristenMcWilliam.Nyrna.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: KristenMcWilliam.Nyrna
+PackageVersion: 2.1.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/Merrit/nyrna/releases/download/v2.1.0/nyrna-windows-installer.exe
+  InstallerSha256: 5F11F3B979E346CE57D27B06AEE3D1BD435D6EE6C72ADB754DE562AEA1B240C4
```
Comment (ImJoakim) [inline manifests/k/KristenMcWilliam/Nyrna/2.1.0/KristenMcWilliam.Nyrna.installer.yaml:10]:
```suggestion
  InstallerSha256: 7672C8659AABDF5D9A54F61BAEF2EFACC1C5AE7459B78DE7FA972896F48353B2
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
New hash: https://github.com/microsoft/winget-pkgs/pull/25052#discussion_r688992066

**Comment by ImJoakim** [comment]
Perfect 😄 

image

**Comment by Merrit** [comment]
Awesome, much appreciate your help! 😀

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**